app/client/src/components/shared/AssessmentSummary/index.js

```diff
@@ -45,6 +45,7 @@ function AssessmentSummary({ waterbodies, fieldName, usageName }: Props) {
     <>
       <div css={modifiedInfoBoxStyles}>
         <p>
+          There {summary.total === 1 ? 'is' : 'are'}{' '}
           <strong>{summary.total.toLocaleString()}</strong>{' '}
           {summary.total === 1 ? 'waterbody has' : 'waterbodies have'} been
           assessed for{' '}
```

Comment:
Good catch! I went ahead and fixed that grammar issue. 

I'm not really a fan of this change either, mainly because of the changes needed in the code.

Reply:
Great! I'd still suggest removing "that has" and "that have" so it's consistent with the wording used elsewhere.

Reply:
Makes sense. I made that update.